src/bindings/rust/README.md

```diff
@@ -0,0 +1,25 @@
+- infos =
+- infos/author = Philipp Gackstatter <philipp.gackstatter@student.tuwien.ac.at>
+- infos/status = experimental maintained
+- infos/provides =
+- infos/description =
+
+# Rust Bindings
+
+Rust bindings for libelektra.
+
+## Generation
+
+Using the `buildelektra-stretch-full` docker image, we have to install cargo and rustc via [rustup](https://rustup.rs).
+We also need to install rusts formattings tools with `rustup component add rustfmt`.
+
+Bindings are generated when buildling the `elektra-sys` crate using `rust-bindgen`.
+
+Additionally, clang is needed (`clang-4.0` worked), otherwise
```

Comment:
how to get this error message? Using an older clang? Or with gcc?

Reply:
I was using the `buildelektra-full` docker image, based on debian stretch and was building with gcc.

Reply:
Or write this section as "troubleshooting". If you get this error:
...
then please install ... to fix the problem.